components/com_kunena/admin/install/install.xml

```diff
@@ -136,6 +136,9 @@
 		<field name="alias"
 		       type="varchar(255)"
 		       null="0"/>
+		<field name="icon"
```

Comment:
Instead of adding a new field in categories table, the setting of icon  should be stored into params field https://github.com/Kunena/Kunena-Forum/blob/develop/components/com_kunena/admin/install/sql/install/mysql/create_tables.utf8.sql#L74